src/pages/ProfilePage/components/EditProfileModal/EditProfileModalComponents/EditProfileSections.tsx

```diff
@@ -298,3 +319,100 @@ export const EditProject = (props: any) => {
     </div>
   )
 }
+
+export const AddProject = (props: {
+  userId: string
+  handleAddProject: (project: ProjectDetails) => Promise<void>
+  handleCancelEdit: () => void
+}) => {
+  const { userId, handleAddProject, handleCancelEdit } = props
+  const [title, setTitle] = useState('')
+  const [description, setDescription] = useState('')
+  const [url, setUrl] = useState('')
+  const [thumbnail, setThumbnail] = useState('')
+
+  console.log('userId:', userId)
+
+  const handleSubmitProject = async (e: React.FormEvent) => {
+    e.preventDefault()
+    console.log('Trying to save project:', title, description)
+    if (!title || !description) {
+      console.log('Title and description are required')
+      return
+    }
+    const newProject: ProjectDetails = {
+      userID: userId,
```

Comment:
running out of time. But are you saying you can pass in empty values if you want to? That's gonna crash this app probably.